Enhance doc

archanaraja opened the pull request:
- Added documentation and code to generate tissue specific consensus peak*counts matrix

archanaraja committed:
Added documentation and code to generate custom consensus peak counts matrix

- The code and documentation allows user to perform custom consensus peak * counts matrix for their tissue of interest

archanaraja committed:
Enhanced documentation

mihirsamdarshi commented on README.md:
```diff
@@ -621,31 +621,55 @@ the [ENCODE ATAC-seq data standards](https://www.encodeproject.org/atac-seq/#sta
 For each of these wrappers, make sure you fill in the appropriate variables at the top of the script before running.
 
-### 6.1. Individual scripts
+#### Individual scripts contained in the wrapper
```
6.1 was removed here, but you retained 6.2 below

archanaraja replied:
This has been fixed

biodavidjm commented on README.md:
```diff
@@ -71,7 +71,7 @@ of environment.
 5. [Flag problematic samples](#5-flag-problematic-samples)
 
-6. [Post-processing scripts](#6-post-processing-scripts)
+6. [Pipeline post-processing](#6-post-processing-scripts)
```
The link doesn't work.

archanaraja replied:
This has been fixed

biodavidjm commented on README.md:
```diff
@@ -621,31 +621,55 @@ the [ENCODE ATAC-seq data standards](https://www.encodeproject.org/atac-seq/#sta
   contigs
 * [merge\_atac\_qc.R](src/merge_atac_qc.R): merge wet lab QC, curated pipeline QC, and alignment stats
+
+### 6.2 Custom analysis
```
I think 6.1 is about processing all tissues simultaneously and 6.2 is for each tissue individually. Something like:

6.1. Generate consensus merged peak and count matrix for all tissues combined
6.2. Generate tissue-specific consensus merged peak and count matrix. 

And please, adjust the text to explain it in that way

archanaraja replied:
All of these suggestions have been incorporated. @biodavidjm

biodavidjm reviewed (commented):
Please, read my comments on the documentation.

archanaraja committed:
Updated the README to address reviewer comments

- Fixed a minor bolding syntax
- Added better explanation and descriptions to generate the consensus peak * tissue matrix across all tissues and individual tissues
- Fixed the dead link

archanaraja committed:
Updated readme

- Modified description for the atac-post-process-wrapper.sh and atac-seq-human-wrapper.sh

archanaraja committed:
Added sections 6.1 and 6.2

archanaraja committed:
Fixed minor typos to script names
